tests/actiontest.cpp

```diff
@@ -169,9 +152,9 @@ private slots:
         QCOMPARE(a1->isChecked(), false);
         QCOMPARE(changeSpy.size(), 2);
     }
-
 };
 
 QTEST_MAIN(ActionTest)
 
+#include "moc_baseprobetest.cpp"
```

Comment:
I think no because there is a macro inside it meaning it can be built 2 differents way.
Or I would have to dissociate both *probe* ways as different classes, then probably I can do that.